.github/CODEOWNERS

```diff
@@ -1,2 +1,7 @@
 # This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @pjfitzgibbons @ps48 @kavithacm @derek-ho @joshuali925 @dai-chen @YANG-DB @rupal-bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az @anirudha @paulstn @sumukhswamy
+*   @opensearch-project/Observability
```

Comment:
Looks like the team doesn't have write access, any workaround better than reverting?